README.md

```diff
@@ -10,17 +10,19 @@ How to use?
 
 Speak is part of the Sugar desktop and is often included.  Please refer to;
 
-* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
+* [Try Online](https://try.sugarizer.org/),
+* [Experience Sugar](https://www.sugarlabs.org/#try),
 * [How to use Sugar](https://help.sugarlabs.org/),
-* [Download Speak using Browse](https://v4.activities.sugarlabs.org/), search for `Speak`, then download, and;
```

Comment:
You have removed my " and;" which signals second last list item, under my Australian Style Guide.